flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/common/CommonPhysicalSink.scala

```diff
@@ -99,11 +105,33 @@ class CommonPhysicalSink (
 
         val operator = new SinkOperator(env.clean(sinkFunction), rowtimeFieldIndex, enforcer)
 
+        val inputParallelism = inputTransformation.getParallelism
+        val taskParallelism = env.getParallelism
```

Comment:
`env.getMaxParallelism`?

Reply:
Well,  correct me if i am wrong, `getMaxParallelism` is the upper bound of parallelism that the task can apply.
`getParallelism ` is the actual parallelism that the task applies

Reply:
Since we don‘t need to do this according to the conversation below here, it is fine to remove this line.